### DISCUSSION TITLE
Lost the ability to use the basic method of showing the icons

### DISCUSSION THREAD

**Discussion by Chronon88** [Ross-Patterson/Portals-Desktop-Organization#60] [General]
I don't like how some icons look when they are not stretched. 
Some of them are too small and stuff because Portals is asking for a high resolution version and this mode does not stretch the icons. I originally fixed this with a setting in a settings menu of portals which seems to be missing now.

On the picture below you can see what I mean. I copied two of the bad looking ones outside the portal and they look fine on Desktop. Inside a Portal not. Fortnite doesn't even have a proper icon to show for the size Portals is asking for and shows some placeholder icon...

![image](https://user-images.githubusercontent.com/4989135/193114668-f4a0e7a7-5000-4076-b875-bd56ebd2c1a5.png)

Please add back the option if it has been removed or let me know how to debug this if the option should still exist and I'm somehow missing it.

**Comment by Ross-Patterson** [comment]
Its definitely still there, its called "Use Alternative Icon Retrieval Method", its next to the icon size dropdown in the settings.

**Reply by Chronon88** [reply]
Oops, I'm blind; thanks! 
I think the color boxes made my brain ignore the options between them >.<

**Reply by Chronon88** [reply]
Btw, should this be the default? I think most people would prefer the way stuff looks when this setting is unticked.
(Portals lost the product key and reverted to default settings for some reason today. Doesn't matter, been working fine for weeks,)

**Comment by Chronon88** [comment]
Hmm. Dunno why  Fortnite icon is still screwed up after unticking the "Use Alternative Icon Retrieval Method"
I just ctrl+c then ctrl+v it out of the Portal here and it seems to work fine on Desktop:

![image](https://user-images.githubusercontent.com/4989135/193119824-9367ddf9-7286-4e14-8c9a-720c56871a2c.png)

Not a big deal because it seems to be just a one-off. Odd though.
